package.json

```diff
@@ -19,10 +19,10 @@
     "@mdx-js/react": "1.6.21",
     "@saucelabs/theme-github-codeblock": "0.0.4",
     "clsx": "1.1.1",
+    "core-js": "3.1.4",
```

Comment:
I don't think you really want to include this file in your PR. I would recommend copying package.json from the master branch and pasting it into your local branch so they match. Then you can commit and repush to your branch and this change will go away.

Reply:
This isn't actually any change--- the order of this object has moved, but it's still in the list of dev dependencies so it's functionally no different than the other one. If this isn't uploaded, it should be in .gitignore, but I can't delete it...